crates/db_schema/src/impls/community.rs

```diff
@@ -431,6 +427,11 @@ impl CommunityFollower {
   }
 }
 
+#[diesel::dsl::auto_type]
+pub fn community_follower_select_subscribed_type() -> _ {
+  community_actions::follow_state.nullable()
+}
```

Comment:
I'd really like to have these on the impl, but unfortunately the current [diesel docs](https://diesel.rs/guides/composing-applications.html) state that: 

> For this specific example the generation of the type definition is suppressed via the no_type_alias option as stable rust does not support associated types for non-trait impls yet.

In other words, you can use `auto_type(no_type_alias)` on an impl function, but then you won't be able to use the auto_type outside of itself, and this has to be top-level.

Its necessary here since a lot of the different views use this select. I wish I had a better place to put it, but I can't think of any.

Reply:
You can leave a todo comment. In general its better to use comments in the code to explain how it works, hat way we can easily find it in the future. Though PR comments also make sense when you specifically explain how something changed.

Reply:
K, I'll do that.